test/distrib/csharp/run_distrib_test.bat

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
We may want to extend the copyright checker to recognize batch files.
Actually we probably want a whitelist of extensions that don't need
copyrights in the copyright checker (eg vcxproj, sln)

On Mon, Feb 29, 2016, 2:39 PM Nathaniel Manista <notifications@github.com>
wrote:

> In test/distrib/csharp/run_distrib_test.bat
> <https://github.com/grpc/grpc/pull/5466#discussion_r54491761>:
>
> > @@ -0,0 +1,21 @@
> > +
>
> What's with a blank line at top-of-file? Should this file not have a
> license header like almost every other in the codebase?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/5466/files#r54491761>.
>